### PR TITLE
KAFKA-14692 Fix some errors and omissions in the migration docs

### PR DESCRIPTION
**Pull request by mumrah**

### CODE REVIEW

File: 34/ops.html [hunk 1]
```diff
@@ -3615,8 +3620,21 @@ <h3>Preparing for migration</h3>
...
     Two things are needed before the migration can begin. The brokers must be configured to support the migration and
-    a KRaft controller quorum must be deployed. For the KRaft deployment, please refer to <a href="#kraft_config">the above documentation</a>.
-    The KRaft controllers should be provisioned with the latest <code>metadata.version</code> of "3.4".
+    a KRaft controller quorum must be deployed. The KRaft controllers should be provisioned with the same cluster ID as
+    the existing Kafka cluster. This can be found by examining one of the "meta.properties" files in the data directories
+    of the brokers, or by running the following command.
+  </p>
+
+  <pre>
+    ./bin/zookeeper-shell.sh localhost:2181 get /cluster/id
```
Comment (mimaison) [inline 34/ops.html]: Text within `<pre>` blocks keep the padding and the final line ending. I find the docs much more readable when we remove the spaces and avoid having a new line at the end.
```
  <pre>
./bin/zookeeper-shell.sh localhost:2181 get /cluster/id</pre>
```

This applies to all other formatted blocks in this section.

**Review by mimaison — APPROVED** [review]
Thanks for the quick fix! I left a suggestions to improve the rendering of formatted blocks. I'll approve so you can merge whenever you're ready.

Can you also open the same PR against kafka:3.4?

File: 34/ops.html (hunk 1 above)
Comment (mumrah) [reply 34/ops.html]: Good call. I'll adjust the other pre-formatted sections as well.

**Comment by mumrah** [comment]
> Can you also open the same PR against kafka:3.4?

Yes, I'll work on that later this week.

**Pull request merged**